cmd/abci-cli/abci-cli.go

```diff
@@ -307,18 +308,36 @@ func cmdCommit(c *cli.Context) error {
 // TODO: Make request and response support all fields.
 func cmdQuery(c *cli.Context) error {
 	args := c.Args()
-	if len(args) != 1 {
-		return errors.New("Command query takes 1 argument")
+	fmt.Println(len(args))
+	if len(args) == 0 {
+		return errors.New("Command query takes 1 or more arguments")
 	}
-	queryBytes, err := stringOrHexToBytes(c.Args()[0])
+
+	queryBytes, err := stringOrHexToBytes(args[0])
 	if err != nil {
 		return err
 	}
+
+	var path = "/store"
+	if len(args) > 1 {
+		path = args[1]
+	}
+
+	var height uint64
+	if len(args) > 2 {
+		height, _ = strconv.ParseUint(args[2], 10, 64)
```

Comment:
No, I mean when user mis inputs something. like `10f` instead of `10`. i.e. checking the result of ParseUInt